src/components/modal/modalColumsCo.js

```diff
@@ -0,0 +1,86 @@
+import React from "react"
+import DialogContent from "@material-ui/core/DialogContent"
+import Divider from "@material-ui/core/Divider"
+import { makeStyles } from "@material-ui/core/styles"
+import theme from "../../theme"
+
+const useStyles = makeStyles({
+  /**
+   * Styles definding the content.With some breakpoints, it behaves
+   * different.
+   */
+  content: {
+    alignContent: "center",
+    display: "flex",
+    justifyContent: "center",
+    [theme.breakpoints.down("md")]: {
+      display: "block",
+    },
+  },
+  /**
+   * Styles for the divider. With some breakpoints, it behaves
+   * different and can even dissapear.
+   */
+  divider: {
+    borderColor: "rgba(209, 209, 209, 0.50)",
+    [theme.breakpoints.up("xs")]: {
+      display: "none",
+    },
+    [theme.breakpoints.up("lg")]: {
+      marginLeft: theme.spacing(8),
+      marginRight: theme.spacing(8),
+      display: "block",
+    },
+    [theme.breakpoints.up("xl")]: {
+      marginLeft: theme.spacing(10),
+      marginRight: theme.spacing(10),
+    },
+
+    borderLeft: "1px solid",
+  },
+})
+
+function ModalColumsCo(props) {
+  const { children } = props
+  const styles = useStyles()
+  let child1
+  let child2
+  /**
+   * Divides the children in order to place them properly
+   */
+  if (Array.isArray(children)) {
+    ;[child1, child2] = children
```

Comment:
I feel confused... what mean `;` 🤔

Reply:
I don't really know, prettier wants it
<img width="544" alt="Captura de Pantalla 2020-06-05 a la(s) 6 42 22 a  m" src="https://user-images.githubusercontent.com/56047417/83872491-b90d2200-a6f7-11ea-891c-75376d5848d5.png">